docs/index.asciidoc

```diff
@@ -20,123 +20,194 @@ include::{include_path}/plugin_header.asciidoc[]
 
 ==== Description
 
-The Dissect filter is a kind of split operation. Unlike a regular split operation where one delimiter is applied to
-the whole string, this operation applies a set of delimiters to a string value. +
-Dissect does not use regular expressions and is very fast. +
-However, if the structure of your text varies from line to line then Grok is more suitable. +
-There is a hybrid case where Dissect can be used to de-structure the section of the line that is reliably repeated and
-then Grok can be used on the remaining field values with more regex predictability and less overall work to do. +
+The Dissect filter plugin extracts unstructured event data into fields using delimiters.
 
-A set of fields and delimiters is called a *dissection*.
+Unlike a regular split operation where one delimiter is applied to the whole
+string, the Dissect operation applies a set of delimiters to a string value. 
 
+The resulting set of fields and delimiters is called a *dissection*.
+
+===== Dissect or Grok? Or both?
+
+Dissect differs from Grok in that it does not use regular expressions and is faster. 
+Dissect works well when data is reliably repeated.
+Grok is a better choice when the structure of your text varies from line to line.
+
+You can use both Dissect and Grok for a hybrid use case when a section of the
+line is reliably repeated, but the entire line is not. The Dissect filter can
+deconstruct the section of the line that is  repeated. The Grok filter can process
+the remaining field values with more regex predictability.
+
+===== Terminology
+
+*dissection* - the set of fields and delimiters remaining after passing through the filter.
```

Comment:
Not strictly true. In grok it is called a "grok pattern" in dissect I called it a dissection, however in the interests of consistency the perhaps "*dissect pattern* - the set of fields and delimiters defining the structure of a textual format, also known as a dissection".